docs/data/sql_functions_zh.yml

```diff
@@ -760,7 +760,9 @@ collection:
   - sql: MAP_FROM_ARRAYS(array_of_keys, array_of_values)
     table: mapFromArrays(array_of_keys, array_of_values)
     description: Returns a map created from an arrays of keys and values. Note that the lengths of two arrays should be the same.
-
+  - sql: MAP_UNION(map1, map2)
+    table: map1.mapUnion(map2)
+    description: 返回一个通过合并两个图 'map1' 和 'map2' 创建的图。这两个图应该有相同的数据结构。如果有重叠的键，'map2' 的值将覆盖 'map1' 的值。如果任一图为空，则返回 null。
```

Comment:
there should be an empty line before next key which is `json`

Reply:
ok